packages/router/src/page-loader.tsx

```diff
@@ -128,8 +132,6 @@ export class PageLoader extends React.Component<Props> {
     // Remove the timeout because the page has loaded.
     this.clearLoadingTimeout()
 
-    this.context.setParams(props.params)
```

Comment:
If you're removing this, you should also remove `static contextType = ParamsContext` further up the file